covid_symptoms/Dockerfile

```diff
@@ -40,17 +37,7 @@ WORKDIR /ctakes
 RUN mvn compile -pl '!ctakes-distribution' -DskipTests
 RUN mvn install -pl '!ctakes-distribution' -DskipTests
 
-WORKDIR /
-RUN mv /ctakes/ctakes-web-rest/target/ctakes-web-rest.war /apache-tomcat-9.0.21/webapps/
-
-RUN rm -rf /root/.m2
-
-ENV TOMCAT_HOME=/apache-tomcat-9.0.21
+FROM tomcat:9.0.70-jre8-temurin
```

Comment:
How about just `tomcat:9-jre8-temurin`? (Just so we silently get non-breaking updates)

Reply:
per our side discussion about 'this was totally fine at my last gig where someone was actively watching for this kind of thing and now it might introduce security vulnerabilities', probably better to just pin to the minor version.